dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -395,6 +395,7 @@
 			<_MonoComponent Include="hot_reload" Condition="'$(MtouchInterpreter)' == 'true'" />
 			<_MonoComponent Include="debugger" Condition="'$(_BundlerDebug)' == 'true'" />
 			<_MonoComponent Include="diagnostics_tracing" Condition="'$(_BundlerDebug)' == 'true'" />
+			<_MonoComponent Include="marshal-ilgen" Condition="'$(_AppleExcludeMarshalIlgenComponent)' != 'true'" />
```

Comment:
Will this get included for the Hot Restart pre-built app or should we add the same to the pre-built .csproj?

Reply:
It will get included automatically.